cli.sh

```diff
@@ -950,7 +949,16 @@ add_shared_hook_repo() {
 
     else
         if ! is_running_in_git_repo_root; then
-            echo "The current directory ($(pwd)) does not seem to be the root of a Git repository!"
+            echo "! The current directory ($(pwd)) does not
+                  seem to be the root of a Git repository!" >&2
+            exit 1
+        fi
+
+        if is_local_url "$SHARED_REPO_URL"; then
+            echo "! Adding an local url:" >&2
```

Comment:
```suggestion
            echo "! Adding a local URL:" >&2
```

Reply:
What is a local URL btw? Is that an absolute path, or in other words NOT a URL?

Reply:
should be `Adding a local path` yes